lib/Gitlab/Api/Environments.php

```diff
@@ -0,0 +1,43 @@
+<?php namespace Gitlab\Api;
+
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class Environments extends AbstractApi
+{
+    /**
+     * @param int $project_id
+     * @return mixed
+     */
+    public function all($project_id)
+    {
+        return $this->get($this->getProjectPath($project_id, 'environments'));
+    }
+
+    /**
+     * @param int $project_id
+     * @param array $parameters
+     * @return mixed
+     */
+    public function create($project_id, array $parameters = array())
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefined('name')
+            ->setAllowedTypes('name', 'string');
+        $resolver->setDefined('slug')
```

Comment:
I don't find any reference in documentation about this `slug` argument. Could you provide some official input that prove it exists? I used to reject parameters not officially documented as those may be painful to maintain in the future.

Reply:
I removed this as I had just misunderstood the return value coming back from the api. It was never a valid parameter. Good catch.